src/components/EventDetails/EventDetails.js

```diff
@@ -18,862 +18,1089 @@
+        }
+        return eventData;
     }
+  }
 
-    if (eventType === COLLECTIBLE_TRANSACTION) {
-      const {
-        to,
-        from,
-        note,
-        gasUsed,
-        gasPrice,
-      } = eventData;
-
-      const isReceived = this.wasTransactionReceived(to);
-      const toMyself = isReceived && addressesEqual(from, to);
-      const fee = gasUsed && gasPrice ? Math.round(gasUsed * gasPrice) : 0;
-      let transactionNote = note;
+  getCollectibleTransactionEventData = (event: Object): EventData => {
+    const { contacts } = this.props;
+    const isReceived = this.isReceived(event);
+    const { asset, icon } = event;
+    const relevantAddress = this.getRelevantAddress(event);
+    const usernameOrAddress = getUsernameOrAddress(event, relevantAddress, contacts);
 
-      if (txNotes && txNotes.length > 0) {
-        const txNote = txNotes.find(txn => txn.txHash === eventData.hash);
-        if (txNote) {
-          transactionNote = txNote.text;
-        }
-      }
-      const hasNote = transactionNote && transactionNote !== '';
-      const recipientContact = this.findMatchingContactOrAccount(to);
-      const senderContact = this.findMatchingContactOrAccount(from);
-      const relatedUser = isReceived ? senderContact : recipientContact;
-      const relatedAddress = isReceived ? from : to;
-      const relatedUserTitle = relatedUser.username
-        // $FlowFixMe
-        || getAccountName(relatedUser.type)
-        || ensRegistry[relatedAddress]
-        || relatedAddress;
-      const relatedUserProfileImage = relatedUser.profileImage || null;
-      // $FlowFixMe
-      const showProfileImage = !relatedUser.type;
+    let eventData: EventData = {
+      name: asset,
+      imageUrl: icon,
+    };
 
-      return (
-        <EventBody>
-          <ListItemUnderlined
-            label={isReceived ? 'SENDER' : 'RECIPIENT'}
-            value={relatedUserTitle}
-            valueAddon={(!!relatedUser.username && <EventProfileImage
-              uri={relatedUserProfileImage}
-              showProfileImage={showProfileImage}
-              userName={relatedUserTitle}
-              diameter={40}
-              initialsSize={fontSizes.regular}
-              style={{ marginBottom: 4 }}
-              onPress={() => this.goToProfile(relatedUser)}
-              noShadow
-              borderWidth={0}
-            />)}
-          />
-          {(toMyself || !isReceived) && !!fee &&
-          <ListItemUnderlined
-            label="TRANSACTION FEE"
-            value={`${utils.formatEther(fee.toString())} ETH`}
-          />
-          }
-          {!!hasNote &&
-          <ListItemParagraph
-            label="NOTE"
-            value={transactionNote}
-          />
-          }
-        </EventBody>
-      );
+    if (isReceived) {
+      eventData = {
+        ...eventData,
+        actionTitle: 'Received',
+        actionSubtitle: `from ${usernameOrAddress}`,
```

Comment:
Based on designs, this line should start with "Collectible".
The one in L780 as well :)